controller/tests/controller_server/utils_test.py

```diff
@@ -28,10 +27,33 @@ def setUp(self):
                                       "publish_context_array_iqn": "array_iqn",
                                       "publish_context_fc_initiators": "fc_wwns"}
                        }
+        self.util_method = None
+
+    def _test_validation_exception(self, method_arg, msg="", raised_error=ValidationException):
+        with self.assertRaises(raised_error) as ex:
+            self.util_method(method_arg)
```

Comment:
I think they are regular functions, not methods

Reply:
done